src/scmdata/run.py

```diff
@@ -356,9 +362,9 @@ def __init__(
         Raises
         ------
         ValueError
-            If metadata for ['model', 'scenario', 'region', 'variable', 'unit'] is not
-            found. A :class:`ValueError` is also raised if you try to load from multiple
-            files at once. If you wish to do this, please use :func:`df_append` instead.
+            * If metadata for ['model', 'scenario', 'region', 'variable', 'unit'] is not found.
+            * If you try to load from multiple files at once. If you wish to do this, please use :func:`scmdata.run.df_append` instead.
```

Comment:
guessing we leave the rename to run.run_append for later?

Reply:
I'll do that in another PR